src/pages/admin-home.tsx

```diff
@@ -12,7 +12,6 @@ import React, {
   SetStateAction,
```

Comment:
We can revert changes to this file as it's part of the old implementation.

Reply:
The changes made to this file is just for re-factoring to the new SearchableContainer component. Let me know if you were referring to something else 🙏